templates/github/.github/workflows/ci.yml.j2

```diff
@@ -91,6 +91,12 @@ jobs:
     runs-on: ubuntu-latest
     # run only after lint finishes
     needs: lint
+    env:
+    {%- if disabled_redis_runners %}
+      REDIS_DISABLED: "{{ disabled_redis_runners | join(' ') }}"
+    {%- else %}
+      REDIS_DISABLED: ""
```

Comment:
I'd rather have a `REDIS_DISABLED=true/false` a few lines down in the matrix.

Some thing like maybe:
`REDIS_DISABLED: "{{ "s3" in (disabled_redis_runners | default([])) }}"`

I think, this makes the generated workflows much more readable. Maybe it's also possible outside of the matrix with
```
env:
  - REDIS_DISABLED: ${{ matrix.env.TEST == "s3" }}
```
This way the workflows would keep their current names.

Reply:
I do not know where else a few lines down in the matrix you could define an environment variable. I think I cannot do this:
```
    strategy:
      fail-fast: false
      matrix:
        env:
          - TEST: pulp
          - TEST: docs
          - TEST: azure
            - REDIS_DISABLED: false
          - TEST: s3
            - REDIS_DISABLED: true
```

Reply:
you could do:
```
    strategy:
      fail-fast: false
      matrix:
        env:
          - TEST: pulp
          - TEST: docs
          - TEST: azure
            REDIS_DISABLED: false
          - TEST: s3
            REDIS_DISABLED: true
```
But on a second thought, this changes the names of the workflow jobs and that'd need adjusting the repo settings everywhere.
So i'd really go with an `env` stanza in the `steps:` part of the workflow.